package Exception;

public class sample4
{
public static void main(String[] args)
{
	
	
	int w []=new int [5];
	
	
	
	try 
	{
		w[6]=100;
		
	}
	
	catch(ArrayIndexOutOfBoundsException b1)
	{
		
	System.out.println("ArrayIndexOutOfBoundsException handled");	
		
	}
	
	System.out.println("akshay");
	
	
	
	
	
}
}
